src/content/structured/community/roadmap.mdx

```diff
@@ -12,22 +12,67 @@ subTitle: "Our roadmap of recently released work and the work we have planned."
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/community/roadmap.mdx"
 ---
 
+## Introduction
+
+The roadmap shows what's new, what we're working on and our future plans.
+
+The purpose of the roadmap is to help teams understand when features are likely to be delivered. Some items might change due to priorities or organisation strategies.
+
+See our [GitHub](https://github.com/orgs/mi6/projects/2/views/1) for more details and our day-to-day activities.
+
 ## What's new
 
-- A public first for MI6, MI5 and GCHQ. We've just launched on [GitHub](https://github.com/mi6).
-- Intelligence Community Design System (ICDS) released as BETA.
-- Research, design and build of nearly 30 web components.
-- Official communications campaign from the United Kingdom Intelligence Community (UKIC).
+We recently added:
+
+- Toasts.
+- Cards V2, with images and expandable area.
+- Asynchronous filtering for search bar.
+- Compact stepper.
+- Chips.
+- Popover menu.
 
```

Comment:
Should these link to the relevant component pages to actually demonstrate they have been released?